packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -1066,20 +1000,57 @@ export class StatsAnalyzer extends EventsScope {
       if (currentPacketsLost < 0) {
         currentPacketsLost = 0;
       }
-
       const currentPacketsReceived =
         result.packetsReceived - this.statsResults[mediaType][sendrecvType].totalPacketsReceived;
       this.statsResults[mediaType][sendrecvType].totalPacketsReceived = result.packetsReceived;
 
       if (currentPacketsReceived === 0) {
         if (receiveSlot && sourceState === 'live') {
           LoggerProxy.logger.info(
-            `StatsAnalyzer:index#processInboundRTPResult --> No packets received for receive slot ${idAndCsi}. Total packets received on slot: `,
+            `StatsAnalyzer:index#processInboundRTPResult --> No packets received for mediaType: ${mediaType}, receive slot ${idAndCsi}. Total packets received on slot: `,
             result.packetsReceived
           );
         }
       }
 
+      if (mediaType.startsWith('video') || mediaType.startsWith('share')) {
```

Comment:
I don't see a similar block for audio, we would need at least the packets received logs for audio

Reply:
The packetsReceived is checked for all mediaTypes, so it doesn't need a specific code block. You can see I send a log for packets received above this codeblock on line 1007